src/System.Net.Primitives/tests/PalTests/System.Net.Primitives.Linux.Pal.Tests.csproj

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The PAL tests for this assembly don't fit neatly into either of these categories:
- They require access to internals, as the PAL is itself internal
- They must by definition make calls to native code, as they are testing exactly the behavior of the PAL, and this behavior relies on the underlying native code

If we're going to be strict about the rule that unit tests may not make calls to native code, then these aren't unit tests; if we're going to be strict about the rule that functional tests may not access internals, then these aren't functional tests. Personally, I'd be more keen on relaxing the latter, but for now I'd prefer to keep the organization as-is.